lib/mina/dsl.rb

```diff
@@ -15,6 +15,15 @@ def invoke(task, *args)
       Rake::Task[task].reenable
     end
 
+    def ensure_git_pushed!(opts = {})
+      branch = opts[:branch] || fetch(:branch)
```

Comment:
Interesting. I was actually thinking of it the other way around - if you write `ensure_git_pushed!` it will do what you expect, and if you have special needs you can specify opts[:branch] or opts[:remote] to override the defaults. Does that sound right?